controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -608,16 +609,30 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
     * @brief Parse AP-Autoconfiguration WSC which should include one AP Radio Basic Capabilities
     * TLV and one WSC TLV containing M1
     */
-    auto radio_basic_caps = cmdu_rx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
+    static const int num_tlvs                                              = 2;
+    std::shared_ptr<wfa_map::tlvApRadioBasicCapabilities> radio_basic_caps = nullptr;
+    std::shared_ptr<ieee1905_1::tlvWscM1> tlvwscM1                         = nullptr;
+    for (int i = 0; i < num_tlvs; i++) {
```

Comment:
There's no reason to have a fixed limit. Just go through all TLVs.

Reply:
OK